testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/actions/AbstractAppInitiatedActionUpdateEmailTest.java

```diff
@@ -57,6 +58,13 @@ public void beforeTest() {
 				.lastName("Doh").build();
 		prepareUser(user);
 		ApiUtil.createUserAndResetPasswordWithAdminClient(testRealm(), user, "password");
+		setRegistrationEmailAsUsername(testRealm(), false);
```

Comment:
Nitpick: Is it possible to remove the change on line  61 and instead call `setRegistrationEmailAsUsername(false);` at the end of the `updateWithEmailAsUsernameEnabled()` test? This is just to make sure that test is not affected if default settings for the realm are changed (For example if emailAsUsername is ever changed to true in the realm).